Makefile

```diff
@@ -0,0 +1,74 @@
+openapi-generator-version:=6.0.1
+openapi-generator-url:=https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/$(openapi-generator-version)/openapi-generator-cli-$(openapi-generator-version).jar
+openapi-generator-jar:=target/openapi-generator-cli.jar
+openapi-generator-cli:=java -jar $(openapi-generator-jar)
+
+generator:=java
+library:=okhttp-gson
+services:=payments management balanceplatform transfers
```

Comment:
These are services we can generate models already. New API's like Management or Balance Platform will be developed soon. Classic Payments have been upgraded in #808. The others below (e.g. recurring, payout, platforms, etc.) will be migrated in separate PR's.